src/components/Dropdown/StyleDropdown.tsx

```diff
@@ -15,16 +35,50 @@ export default function Styledropdown() {
       setIcon(<BiPlus size={18} />);
     }
   }
+
+  // this functin can be removed any timee, just for testing purposes
+  const handleSeed = async () => {
+    seed.forEach(async (el) => {
+      await DataStore.save(
+        new Category({
+          ...el,
+        })
+      );
+    });
+  };
+
+  // this functin can be removed any timee, just for testing purposes
+  const handleDeleteSeed = async () => {
+    await DataStore.delete(Category, (category) =>
+      category.name.contains("test")
+    ); // delete all the posts
+  };
   return (
     <div className="dropdown">
+      {/* div and two buttons two buttons can be removed at any time, just for testing purposes */}
```

Comment:
Thanks for doing this. Made testing much easier. Before merging, please delete the seeding functionality